translations/models.py

```diff
@@ -87,23 +91,23 @@ class Translation(TranslatableModel):
     )
     active = models.BooleanField(default=True, null=False)
     no_auto = models.BooleanField(default=False, null=False)
-    label = models.CharField(max_length=128, null=False, blank=False, unique=True)
+    label = models.CharField(max_length=128, null=False,
+                             blank=False, unique=True)
 
     objects = TranslationManager()
 
-    def get_lang(self, lang):
-        return self.translations.filter(language_code=lang).first()
-
-    def in_program(self):
+    def find_used_model(self, label_unpack=False):
+        has_relationship = False
         # determine if a translation is refrenced by either a program, navigator, urgent_need, or document
         # https://stackoverflow.com/questions/54711671/django-how-to-determine-if-an-object-is-referenced-by-any-other-object
-        has_relationship = False
         for reverse in (f for f in self._meta.get_fields() if f.auto_created and not f.concrete):
             if reverse.related_name == 'translations':
                 continue
             name = reverse.get_accessor_name()
             has_reverse_other = getattr(self, name).count()
             if has_reverse_other:
+                if label_unpack:
+                    return reverse.related_model._meta.model_name
                 try:
                     active = getattr(self, reverse.related_name).first().active
```

Comment:
@msrezaie Here is an example of where you get a property on the reverse model. You can modify this to get fields you need instead of parsing the regex.